recipes/pysd/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "pysd" %}
+{% set version = "1.5.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pysd-{{ version }}.tar.gz
+  sha256: d6c759a8a64ff3a7df939c9e4aaba57cf6bbffbdf46f1bc1dc14b3c3c59b2860
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
```

If noarch python, then this should be:
```suggestion
    - python >=3.7
```